option optimize_for = LITE_RUNTIME;

message GetPlayerAttrRequest
{
 repeated uint32 attr_id = 1;
}
